pkg/ccl/crosscluster/logical/logical_replication_job_test.go

```diff
@@ -493,6 +493,34 @@ func TestRestoreFromLDR(t *testing.T) {
 	dbC.CheckQueryResults(t, "SELECT * FROM tab", [][]string{{"1", "hello", "2"}})
 }
 
+func TestImportIntoLDR(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	skip.UnderDeadlock(t)
+	defer log.Scope(t).Close(t)
+
+	ctx := context.Background()
+	dataDir, dirCleanupFunc := testutils.TempDir(t)
+	defer dirCleanupFunc()
+	args := testClusterBaseClusterArgs
+	args.ServerArgs.ExternalIODir = dataDir
+	server, s, dbA, dbB := setupLogicalTestServer(t, ctx, args, 1)
```

Comment:
Kinda kills me to spin up a whole test server and job just to test a single if statement during a job planning.

Is there an existing LDR test of allowed/disallowed flags or one of schema locking to which we could add one more case to check?